.github/workflows/main.yml

```diff
@@ -0,0 +1,405 @@
+name: Build & Test
+
+on:
+  # Trigger on push to branches `master` and `3.5`.
+  push:
+    branches: [ master, 3.5 ]
+  # Trigger on pull request events for PRs that have `master` as their target branch
+  pull_request:
+    branches: [ master ]
+
+  # Allow running the workflow manually
+  workflow_dispatch:
+
+defaults:
+  run:
+    # GitHub Actions adds `errexit` and `pipefail` by default, but we add `xtrace`
+    # to improve debugging some of the longer scripts.
+    shell: /bin/bash -o errexit -o pipefail -o xtrace {0}
+
+jobs:
+  main:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-20.04
```

Comment:
same question.

```suggestion
    runs-on: ubuntu-latest
```

Reply:
Yes, it was sorted in the other comments thread. I'd like to mark these threads as resolved, but it seems only @sclu1034 can do. The only option I had was to hide my comment, and it has only hid my comment, not the full thread...